.eslintrc.json

```diff
@@ -26,7 +26,8 @@
         "eol-last": ["error", "always"],
         "no-console": ["error", { "allow": ["warn", "error"] }],
         "no-use-before-define": ["off"],
-        "react/display-name": ["off"]
+        "react/display-name": ["off"],
+        "jest/expect-expect": ["off"]
```

Comment:
This is intentional but we can talk about it. The way puppeteer works, it will time out and throw an error when expected UI elements fail to appear. In a sense every line of an e2e test can be considered an assertion. That means explicit assertions would be both redundant ... and unreachable!

It's true that the current e2e tests all have assertions in them, but I can 100% envision valid e2e tests that don't include them, that's why I removed the requirement.

Does that clear things up? Any follow up comments?

Reply:
Interesting! I'm not blocking on this but here are some follow up thoughts that could be discussed later on:
1. While I'm not the biggest fan of using `warn`. This does seem like a case that _could_ warrant it.
2. You could use jest's [overrides](https://eslint.org/docs/latest/use/configure/rules#using-configuration-files-1) to turn that rule off for an e2e test. But it would mean having to now name e2e test suites with a specific name. eg. `AtVersion.test.js` -> `AtVersion.e2e.test.js`
3. Ignoring each e2e file instead.

This is all opinion though, really I don't expect tests outside of the e2e ones as you've explained, to have missing expects anyways and even then, would be caught by a review most likely. But purely a precaution.

Reply:
I'll revert the change then.